spipe/main.c

```diff
@@ -306,29 +326,33 @@ main(int argc, char * argv[])
 	if (ET.connection_error)
 		goto err0;
 
+	sock_addr_freelist(sas_b);
```

Comment:
Shouldn't this be in the "Clean up" section, on line 324?  (i.e. move it 5 lines higher)

Reply:
Fixed.